lib/tasks/metrics.rake

```diff
@@ -3,6 +3,9 @@ namespace :metrics do
   task overview: :environment do
     puts "Admin Overview Link Tracking for #{SiteConfig.app_domain}:"
     links_by_target = Ahoy::Event.overview_link_clicks.group("properties -> 'target'").count
+
+    return unless links_by_target
```

Comment:
I added this after trying to write a test for this and getting a very odd `cannot call delete_prefix on nil` error, after which I realized that we probably don't want or even need to try to iterate over `links_by_target` if there aren't any links to iterate over 🤷🏽